python/ray/rllib/agent.py

```diff
@@ -252,6 +252,9 @@ def get_agent_class(alg):
         return _SigmoidFakeData
     elif alg == "__parameter_tuning":
         return _ParameterTuningAgent
+    elif alg == "DDPG":
+        from ray.rllib import ddpg
+        return ddpg.DDPGAgent
```

Comment:
Along with registering it here, could you also register DDPG in these automated tests?

1) The generic test for supported obs/action spaces:
https://github.com/ray-project/ray/blob/master/python/ray/rllib/test/test_supported_spaces.py#L132

2) The regression tests folder: https://github.com/ray-project/ray/tree/master/python/ray/rllib/tuned_examples/regression_tests

3) The checkpoint/restore test https://github.com/ray-project/ray/blob/master/python/ray/rllib/test/test_checkpoint_restore.py

You should be able to basically copy paste an existing entry for e.g., DQN in each of these cases.